Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -1723,6 +1723,14 @@ def generate_webkit_secure_coding_impl(serialized_types, headers):
             header_set.add(header)
     result.extend(output_sorted_headers(sorted(header_set)))
 
+    needDictionaryForWebKitSecureCodingType = False
+    for type in serialized_types:
+        if not type.is_webkit_secure_coding_type():
+            continue
+        useWKKeyedCoderOnly = type.support_wkkeyedcoder and type.custom_secure_coding_class is None
```

Comment:
This doesn't seem right.  We'd need to check the conditions of each type instead, which would require lots of complexity.  Why don't we instead just add [[maybe_unused]]?